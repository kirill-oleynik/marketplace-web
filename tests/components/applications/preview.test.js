const React = require('react');
const { shallow } = require('enzyme');
const toJSON = require('enzyme-to-json').default;
const ApplicationPreview = require('../../../app/components/applications/preview').default;

describe('#render', () => {
  test('it renders correctly', () => {
    const component = shallow(
      <ApplicationPreview
        application={{
          id: 1,
          title: 'Test1',
          description: 'Hello World!',
          logo: 'https://example.com/image.png'
        }}
      />
    );

    expect(
      toJSON(component)
    ).toMatchSnapshot();
  });
});
