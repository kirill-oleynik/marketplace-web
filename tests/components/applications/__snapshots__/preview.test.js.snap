// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render it renders correctly 1`] = `
<span
  className="app-item"
>
  <div>
    <span
      className="app-item__img mb-5 mr-25"
      style={
        Object {
          "backgroundImage": "url('https://example.com/image.png')",
        }
      }
    />
  </div>
  <div>
    <div
      className="d-flex justify-content-between"
    >
      <h4
        className="app-item__title"
      >
        Test1
      </h4>
      <button
        className="app-item__remove mb-5"
        type="button"
      >
        <i
          className="app-item__icon icon icon-cross font-8"
        />
      </button>
    </div>
    <p
      className="app-item__text"
    >
      Hello World!
    </p>
  </div>
</span>
`;
